libs/bower/bower.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "ilias",
+  "homepage": "https://ilias.de",
+  "authors": [
+    "ILIAS <info@ilias.de>"
+  ],
+  "description": "",
+  "main": "",
+  "license": "GPL-3.0",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "jquery": "3.2.1",
+    "bootstrap": "3.3.7",
+    "jquery-ui": "1.12.0"
+  },
+  "extra": {
+    "jquery": {
+      "introduction-date": "unknown",
```

Comment:
Is there a `null` or `undefined` in JSON? Let's use that instead of a magic string then.